pkg/cache/memory/draft.go

```diff
@@ -68,3 +68,13 @@ func (cd *cachedDraft) Close(ctx context.Context, version string) (repository.Pa
 		return cd.cache.update(ctx, closed)
 	}
 }
+
+func (cd *cachedDraft) refreshCache(ctx context.Context) error {
```

Comment:
I don't like this placement of a cache refresh. Ordering a full cache refresh is extremely